docs/reference/crds/certificate-stores.md

```diff
@@ -25,7 +35,18 @@ spec:
           certificateName: yourCertName
           certificateVersion: yourCertVersion 
     tenantID:
-    clientID: 
+    clientID:
+status:
+  issuccess:        true
+  lastfetchedtime:  2023-04-06T18:50:37Z
+  properties:
+    certificates:
+      certificate Name:  wabbit-networks-io
+      last Refreshed:    2023-04-06T18:50:37Z
+      version:           97a1545d893344079ce57699c8810590
+      certificate Name:  wabbit-networks-io
+      last Refreshed:    2023-04-06T18:50:37Z
+      version:           72086d944ff04bf2bcbf36cb33728a86
 ```
```

Comment:
wonder if we need to mention that users could run `kubectl get` to get a quick review?